Composer/packages/client/src/pages/publish/index.tsx

```diff
@@ -310,7 +311,7 @@ const Publish: React.FC<PublishPageProps> = props => {
         await actions.publishToTarget(projectId, selectedTarget, { comment: comment }, sensitiveSettings);
 
         // update the target with a lastPublished date
-        const updatedPublishTargets = settings.publishTargets.map(profile => {
+        const updatedPublishTargets = settings.publishgargets.map(profile => {
```

Comment:
type

Reply:
Nice catch. I'm going to update the settings type so this will be caught.